Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG PYTHON_VERSION=3.6
```

Comment:
I'd say it is better to place this file in docker directory though

Reply:
any reasons why?  
I see most projects placed their main dockerfile in the main directory, and if they have more dockerfile they add in the docker directory then

Reply:
the main reason is that it seems like single dockerfile wont be sufficient in the future. As for other projects, there are also projects that put dockerfiles in separate directory: 

https://github.com/microsoft/onnxruntime/tree/master/dockerfiles
https://github.com/cupy/cupy/tree/master/docker/python3
https://github.com/sony/nnabla/tree/master/docker
https://github.com/apache/incubator-mxnet/tree/master/docker

so i dont think putting dockerfile in the root dir is the best option

Reply:
yes that's what I mean, even if this main dockerfile is not sufficient, we could still add additional files in the docker directory. I put it in the main directory just to make it easier to find them.

Reply:
so in your opinion putting dockerfiles in separate directory instead of root dir is harder to find? hmm okay